src/AaveV3Update_20231002weth/AaveV3Update_20231002weth.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Gauntlet Recommendations to Lower WETH Variable Base to 0 on Arbitrum, Optimism v3
+author: Paul Lei, Watson Fu, Nick Del Zingaro, Walter Li, Jonathan Reem, Nick Cannon, Sarah Chen, Dana Tung
```

Comment:
idk how reasonable it is to put all the names here instead of just "Gauntlet", as this is how it will render on the most popular governance interface 
![image](https://github.com/bgd-labs/aave-proposals/assets/4396533/403bf21e-fc01-44ed-9701-707028dbe436)